pkgs/development/libraries/nghttp2/default.nix

```diff
@@ -6,6 +6,7 @@
 , enableAsioLib ? false, boost ? null
 , enableGetAssets ? false, libxml2 ? null
 , enableJemalloc ? false, jemalloc ? null
+, enableApp ? false
```

Comment:
I was trying to follow the derivation as it was as closely as possible but I'll admit I hadn't thought of your suggestion :smile:.

I'll follow your suggestion and push a revision to the commit this evening. Thanks @fpletz.